tools/ci_build/github/apple/c/onnxruntime-mobile-c.config.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name": "onnxruntime-mobile-c",
+    "summary": "ONNX Runtime Mobile C/C++ Pod",
+    "description": "A pod for the ONNX Runtime Mobile C/C++ library."
+}
```

Comment:
Can we explicitly mention this involves a reduced set of opsets/ops/types and only supports ORT format models?